.github/copilot-instructions.md

```diff
@@ -0,0 +1,212 @@
+# Copilot Instructions for Compose Multiplatform HTML Unified
+
+## Repository Overview
+
+**Compose Multiplatform HTML Unified** is a Kotlin Multiplatform library that provides unified Compose Multiplatform wrappers of common and Material Design APIs for both **rendering-based Compose UI** (Android, desktop JVM, iOS, web Kotlin/Wasm) and **DOM-based Compose HTML**. This library was previously named "Compose Multiplatform Material".
+
+### Repository Statistics
+- **Size**: Medium-large multiplatform project with ~15 modules
+- **Language**: Kotlin (multiplatform)
+- **Build System**: Gradle with Kotlin DSL
+- **Target Platforms**: JVM, Android, iOS, JS (DOM), Wasm-JS (Canvas)
+- **Key Dependencies**: Compose Multiplatform, Kobweb Silk, Material Web Components
+- **Current Version**: 0.6.0-SNAPSHOT
```

Comment:
Is it necessary to mention the version here? This version is constantly updated.